geth/account/proxy.go

```diff
@@ -0,0 +1,32 @@
+package account
+
+import (
+	"context"
+	"github.com/status-im/status-go/geth/common"
+	"github.com/status-im/status-go/geth/rpc"
+)
+
+// AccountsRPCHandler returns RPC Handler for the Accounts() method.
+func (m *Manager) AccountsRPCHandler() rpc.Handler {
+	return func(context.Context, ...interface{}) (interface{}, error) {
+		return m.Accounts()
+	}
+}
+
+func newAccountNodeManager(node common.NodeManager) *accountNodeManager {
+	return &accountNodeManager{node: node}
+}
+
+type accountNodeManager struct {
```

Comment:
We were actually removing a bunch of `-Managers` for a last couple of month, and it's definitely not that pattern to copy when introducing the new logic.

Reply:
There are no new logic. Only tests and 3 methods extraction. So, there are manager in account package and node manager from node package using here. Renaming this managers is separated task.